### PR TITLE
Add styleguide for buttons

### LINKED ISSUE #888: Styleguide: Buttons (Simple)
Document buttons used within the app in a styleguide section

Includes:

- default
- primary
- cancel
- submit

Also:

- `<Section name="buttons" title="Buttons"></Section>`
- define a `Buttons.scss`
- define a `Buttons.js` in `/styleguide/components/sections`

Styleguide Index: #884
reference #877

### PR DESCRIPTION
**Pull request by briansteffens**
Closes #888

screen shot 2017-01-13 at 5 23 27 pm

### CODE REVIEW

File: src/linodes/components/ConfigSelectModalBody.js [hunk 1]
```diff
@@ -43,21 +45,20 @@ export default class ConfigSelectModalBody extends Component {
...
-            className="btn btn-cancel"
+          <CancelButton
             disabled={loading}
             onClick={() => dispatch(hideModal())}
-          >Cancel</button>
-          <button
-            className="btn btn-default"
+          />
+          <Button
+            className="LinodesLinodeComponentsConfigSelectModalBody-submit"
```
Comment (abemassry) [inline src/linodes/components/ConfigSelectModalBody.js]: 👍

File: scss/components/buttons.scss [hunk 2]
```diff
@@ -0,0 +1,153 @@
+//@import 'variables';
```
Comment (mparke) [inline scss/components/buttons.scss]: dead comments?

File: scss/linodes/linode/layouts/RebuildPage.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 .LinodesLinodeRebuildPage {
-    &-rebuild {
-        @extend .btn;
-        @extend .btn-default;
-    }
+//    &-rebuild {
```
Comment (mparke) [inline scss/linodes/linode/layouts/RebuildPage.scss]: dead comment?

File: scss/linodes/linode/settings/components/DiskPanel.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.DiskPanel-edit {
+    margin-right: 0.5rem;
```
Comment (mparke) [inline scss/linodes/linode/settings/components/DiskPanel.scss]: use pixel values for padding/margin, rem/em should be reserved for text going forward.

**Pull request merged**